### PR TITLE
updated welcome string spanish

### PR DESCRIPTION
**Pull request by ericaeilenberg**

### CODE REVIEW

**Comment by francisli** [comment]
Great! I'll review later today...

File: lib/strings.js [hunk 1]
```diff
@@ -284,8 +284,8 @@ export default Strings = new LocalizedStrings({
     about: {
       what: '什麽是Intentional Walk?',
       whatText: 'Intentional Walk 是一項於2020年7月1日至7月31日免費運行的社區步行計劃.該計劃向有資格參加CalFresh / Medical的舊金山居民開放.',
-      dates: 'What are the key dates?',
-      datesText: 'On July 1, your steps will start at zero and the walking challenge will begin! All of your steps from July 1-30 will count towards winning a prize. After the program is over, you can use the app, but no additional prizes will be awarded.',
+      dates: '關鍵日期?',
+      datesText: '挑戰从7月1日起, 您的步行数將從零開始! 7月1日至31日, 您所有的步行数都將計入獎品中. 計劃結束後, 您可以使用該應用程序，但不會獲得其他獎品.',
```
Comment (francisli) [inline lib/strings.js:288]: @ericaeilenberg you changed the end date here in Chinese to 31, but in English and Spanish, it still says we only track steps until 7/30...?

Comment (ericaeilenberg) [reply lib/strings.js:288]: The Chinese is actually correct! Should be the entire month of July, 1-31

Comment (francisli) [reply lib/strings.js:288]: Ok, I'll make that last change in English and Spanish myself and will merge in all the changes...

**Pull request merged**